src/app/screens/Accounts/Show/index.tsx

```diff
@@ -0,0 +1,592 @@
+                  <Trans
+                    i18nKey={"nostr.private_key.subtitle"}
+                    t={t}
+                    components={[
+                      // eslint-disable-next-line react/jsx-key
+                      <a
+                        className="underline"
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://guides.getalby.com/overall-guide/alby-browser-extension/features/nostr"
+                      ></a>,
+                    ]}
+                  />
+                </p>
+              </div>
+              <div className="w-1/5 flex-none ml-6">
+                <Button
+                  label={t("nostr.actions.generate")}
+                  onClick={() => setNostrKeyModalIsOpen(true)}
+                  fullWidth
+                />
+              </div>
+            </div>
+            <div className="mb-4 flex justify-between items-end">
+              <div className="w-7/12">
+                <TextField
+                  id="nostrPrivateKey"
+                  label={t("nostr.private_key.label")}
+                  type={nostrPrivateKeyVisible ? "text" : "password"}
+                  value={nostrPrivateKey}
+                  onChange={(event) => {
+                    setNostrPrivateKey(event.target.value);
+                  }}
+                  endAdornment={
+                    <button
+                      type="button"
+                      tabIndex={-1}
+                      className="flex justify-center items-center w-10 h-8"
+                      onClick={() => {
+                        setNostrPrivateKeyVisible(!nostrPrivateKeyVisible);
+                      }}
+                    >
+                      {nostrPrivateKeyVisible ? (
+                        <HiddenIcon className="h-6 w-6" />
+                      ) : (
+                        <VisibleIcon className="h-6 w-6" />
+                      )}
+                    </button>
+                  }
+                />
+              </div>
+              <div className="w-1/5 flex-none mx-4">
+                <Button
+                  icon={<CopyIcon className="w-6 h-6 mr-2" />}
+                  label={privateKeyCopyLabel}
+                  onClick={async () => {
+                    try {
+                      navigator.clipboard.writeText(nostrPrivateKey);
+                      setPrivateKeyCopyLabel(tCommon("copied"));
+                      setTimeout(() => {
+                        setPrivateKeyCopyLabel(tCommon("actions.copy"));
+                      }, 1000);
+                    } catch (e) {
+                      if (e instanceof Error) {
+                        toast.error(e.message);
+                      }
+                    }
+                  }}
+                  fullWidth
+                />
+              </div>
+              <div className="w-1/5 flex-none">
```

Comment:
non-blocking: Would be nice to have this in a `form` so I can use "enter" to submit after pasting my nsec.